acceptance/tests/resource/group/should_query_all.rb

```diff
@@ -9,6 +8,15 @@
   skip_test('this test fails on windows French due to Cygwin/UTF Issues - PUP-8319,IMAGES-492') if agent['platform'] =~ /windows/ && agent['locale'] == 'fr'
   step "query natively"
 
+  # [PA-4555] Added below code to enable SSH permissions before test starts if they are disabled by default
+  on(agent, 'dscl . list /Groups | grep com.apple.access_ssh') do
+    stdout.each_line do |line|
+      if line =~ /com.apple.access_ssh-disabled/
+        on(agent, 'dscl . change /Groups/com.apple.access_ssh-disabled RecordName com.apple.access_ssh-disabled com.apple.access_ssh')
+      end
+    end
+  end
+
```

Comment:
Just adding a note based on my standup comments. Feel free to move `PA-4555` to the PUP project and wrap this block if code in a conditional for osx ARM hosts.